gnovm/cmd/gno/testdata/test/flag_failfast.txtar

```diff
@@ -0,0 +1,30 @@
+# Test -failfast flag
+
+! gno test -failfast ./...
+
+! stdout .+
+stderr '--- FAIL: TestFail'
+stderr 'someError'
+! stderr 'secondError'
```

Comment:
also test that a second top-level function isn't executed

Reply:
is this ok ?
cb4ee65fa8a42d5fc01fe79be1660b1367c0d937